### ISSUE TITLE
is it the correct version?

### ISSUE THREAD

**Issue by Zealoe** [Zardinality/TF-deformable-conv#3]
is it the correct version of deformable conv? and as for the training details, do you have some advices? for examples,learning rate？

**Comment by Zardinality** [comment]
I guess so, you can compare the result with the original version deform conv in mxnet. I have tested it with some random input. I would be appreciated if you find some counter examples. Although the cuda code are the same, the tensorflow wrap is kind of messy so I do not make 100 percent guarantee.
I remember the original author made some comments about the training. But I don't have much experiences tuning it, sorry.
